mocap_optitrack_driver/config/mocap_optitrack_driver_params.yaml

```diff
@@ -1,12 +1,12 @@
 mocap_optitrack_driver_node:
   ros__parameters:
     connection_type: "Unicast" # Unicast / Multicast
-    server_address: "198.17.0.120"
```

Comment:
Please, remove this file from your commit. This change is related to your configuration.

Reply:
Yes agreed. Sorry for the clutter.